src/main/java/com/cloudbees/jenkins/plugins/sshagent/mina/MinaRemoteAgentStarter.java

```diff
@@ -24,15 +24,23 @@
 
 package com.cloudbees.jenkins.plugins.sshagent.mina;
 
+import jenkins.security.MasterToSlaveCallable;
+
 import com.cloudbees.jenkins.plugins.sshagent.RemoteAgent;
+
 import hudson.model.TaskListener;
-import hudson.remoting.Callable;
 import hudson.remoting.Channel;
 
 /**
  * Callable to start the remote agent.
  */
-public class MinaRemoteAgentStarter implements Callable<RemoteAgent, Throwable> {
+public class MinaRemoteAgentStarter extends MasterToSlaveCallable<RemoteAgent, Throwable> {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 2909967505958103050L;
```

Comment:
@stephenc I'm bit confused. In master, [this class](https://github.com/jenkinsci/ssh-agent-plugin/blob/master/src/main/java/com/cloudbees/jenkins/plugins/sshagent/mina/MinaRemoteAgentStarter.java#L35) does not have UID. Where is the problem here?